Week1/Solution/3.palindram or not_1.py

```diff
@@ -0,0 +1,12 @@
+#write a python script to enter any number and check it is palindram number or not.
+n=int(input('Enter Your Number: '))
+temp=n
+s=0
+while n>0:
+    r=n%10
+    s=r+(s*10)
+    n=n//10

```

Comment:
why //?

Reply:
In the program, the // operator is used for integer division. 
The main purpose of using this operator to perform floor division,which gives the value without any reminder.

Reply:
VERY GOOD...!!!